cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java

```diff
@@ -175,24 +187,35 @@ void printMessage(String topic, MqttMessage message) {
         }
     }
 
-    static MqttConnectOptions setConnectionOptions(MqttConnectOptions connectOptions) {
-        connectOptions.setUserName("admin");
-        connectOptions.setPassword("admin".toCharArray());
+    static MqttConnectOptions setConnectionOptions(MqttConnectOptions connectOptions, String username, String password) {
+
+        if (!username.isEmpty()) {
+            connectOptions.setUserName(username);
+            if (password != null) {
+                connectOptions.setPassword(password.toCharArray());
+            }
+        }
+
         connectOptions.setCleanSession(true);
 
         return connectOptions;
     }
 
     void closeClient(MqttClient client) throws MqttException {
         if (client != null) {
-            client.disconnect();
-            client.close();
+            try {
+                client.disconnect();
+                client.close();
+            } catch (MqttException e) {
+                client.close();
+                throw e;
+            }
         }
     }
 
     protected Logger setUpLogger(String name) {
         Logger log = Logger.getLogger(name);
-        log.setLevel(Level.INFO);
+        log.setLevel(Level.WARN);
```

Comment:
We will need an option to have logging enabled from commandline. Can you create a Jira for yourself (or github issue, whatever, just so we dont forget) and assign it to yourself?

Reply:
MSGQE-6001 Create option for enabling logging from command line in cli-paho-java